R/quosure.R

```diff
@@ -211,6 +217,13 @@ quo_label <- function(quo) {
 quo_text <- function(quo, width = 60L, nlines = Inf) {
   expr_text(quo_expr(quo), width = width, nlines = nlines)
 }
+#' @export
+quo_name <- function(quo) {
+  if (!is_symbol(quo)) {
+    abort("`quo` must quote a symbol")
```

Comment:
I'm not entirely sure. Something like:

* if name, convert to character
* if call, do something like `expr_label()` but wrap with `()`
* if atomic scalar, convert to character

Anything else can be an error